iogt/templates/admin/import_from_git.html

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
You don't need to extract this file since it's not changing. Let's leave it out.